app/src/main/java/com/movies/common/extensions/ViewExtension.kt

```diff
@@ -15,24 +19,27 @@ import androidx.core.content.ContextCompat
 import com.movies.presentation.view.search_and_filter.adapter.model.CategoryStyle
 
 fun View.visibleIf(visible: Boolean) {
+    visibility = if (visible) View.VISIBLE else View.INVISIBLE
+}
+
+fun View.visibleIfWithAnimation(visible: Boolean) {
+    animation = AnimationUtils.loadAnimation(
+        context,
+        androidx.appcompat.R.anim.abc_grow_fade_in_from_bottom
+    )
     visibility = if (visible) View.VISIBLE else View.GONE
 }
```

Comment:
just make one function, and pass boolean type into the param, is that param will be true set animation